tests/robustness/traffic.go

```diff
@@ -133,6 +134,61 @@ type requestChance struct {
 	chance    int
 }
 
+type kubernetesTraffic struct {
+	keyCount int
+}
+
+func (t kubernetesTraffic) Run(ctx context.Context, clientId int, c *recordingClient, limiter *rate.Limiter, ids identity.Provider, lm identity.LeaseIdStorage, finish <-chan struct{}) {
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case <-finish:
+			return
+		default:
+		}
+		resource := "pods"
+
+		pods, err := t.Range(ctx, c, "/registry/"+resource+"/", true)
+		if err != nil {
+			continue
+		}
+		limiter.Wait(ctx)
+		if len(pods) < t.keyCount {
+			err = t.Create(ctx, c, "/registry/"+resource+"/default/"+stringutil.RandString(5), fmt.Sprintf("%d", ids.RequestId()))
+			continue
+		} else {
+			randomPod := pods[rand.Intn(len(pods))]
+			err = t.Update(ctx, c, string(randomPod.Key), fmt.Sprintf("%d", ids.RequestId()), randomPod.ModRevision)
+		}
+		if err != nil {
+			continue
+		}
+		limiter.Wait(ctx)
+	}
+}
+
+func (t kubernetesTraffic) Range(ctx context.Context, c *recordingClient, key string, withPrefix bool) ([]*mvccpb.KeyValue, error) {
+	ctx, cancel := context.WithTimeout(ctx, RequestTimeout)
+	resp, err := c.Range(ctx, key, withPrefix)
+	cancel()
+	return resp, err
+}
+
+func (t kubernetesTraffic) Create(ctx context.Context, c *recordingClient, key, value string) error {
+	ctx, cancel := context.WithTimeout(ctx, RequestTimeout)
+	err := c.CompareAndSet(ctx, key, value, 0)
+	cancel()
+	return err
+}
```

Comment:
It almost has the same implementation as `(kubernetesTraffic) Update`, it seems that we can just remove it, and use `Update` directly by passing a `0` expectedRevision.